README.md

```diff
@@ -100,3 +112,10 @@ To cite this lesson, please consult with [CITATION](CITATION)
 [episode08]: https://swcarpentry.github.io/python-novice-inflammation/08-defensive/index.html
 [episode09]: https://swcarpentry.github.io/python-novice-inflammation/09-debugging/index.html
 [episode10]: https://swcarpentry.github.io/python-novice-inflammation/10-cmdline/index.html
+[community-initiatives]: https://communityin.org
+[cp-about]: https://carpentries.org/about
+[swc-about]: https://software-carpentry.org/about/
+[swc-lessons]: https://software-carpentry.org/lessons/
```

Comment:
Is this needed?

Reply:
Nice catch! I used it somewhere in "About SWC section" but either didn't commit it or removed the sentence with it (don't remember). What do you think -- should we add this link somewhere or should we not?

Reply:
I don't think it is necessary to include the link on the README, since it is linked near the top of the software-carpentry.org/about/ page to which we do link, but if you feel like it should be linked, I think sticking it back into the _About Software Carpentry_ section makes most sense.

Reply:
Link removed!